lib/bottom_navy_bar.dart

```diff
@@ -135,29 +180,30 @@ class _ItemWidget extends StatelessWidget {
               mainAxisAlignment: MainAxisAlignment.start,
               crossAxisAlignment: CrossAxisAlignment.center,
               children: <Widget>[
-                IconTheme(
-                  data: IconThemeData(
-                    size: iconSize,
-                    color: isSelected
-                        ? item.activeColor.withOpacity(1)
-                        : item.inactiveColor == null
-                            ? item.activeColor
-                            : item.inactiveColor,
+                Container(
+                  child: FittedBox(
+                    child: Opacity(
+                      opacity: isSelected ? 1.0 : inactiveOpacity,
+                      child: item.icon,
+                    ),
                   ),
-                  child: item.icon,
                 ),
                 if (isSelected)
                   Expanded(
                     child: Container(
-                      padding: EdgeInsets.symmetric(horizontal: 4),
-                      child: DefaultTextStyle.merge(
-                        style: TextStyle(
-                          color: item.activeColor,
-                          fontWeight: FontWeight.bold,
+                      padding: EdgeInsets.symmetric(
+                          vertical: verticalItemPadding,
+                          horizontal: horizontalItemPadding),
```

Comment:
Hi @pawelpetruch 
Thanks for this amazing contibution. Those two proerties could just be a Padding object so that users will have the hability to change a veri specific padding (left only, right only, top only, etc..)

Thank you

Sorry for the later feedback!